### PR TITLE
added 5 actions around aws service quotas

### PR DESCRIPTION
**Pull request by dougsillars**
added 5 actions around service quotas in AWS:

### CODE REVIEW

File: AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    quota_code: Optional[str] = Field(
+        '"L-4FB7FF5D"',
```
Comment (jayasimha-raghavan-unskript) [inline AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py:11]: Should we have a default value? Since it is an optional field?

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
Please take care of small comment.

File: AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py (hunk 1 above)
Comment (dougsillars) [reply AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py:11]: its not optional

File: AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    quota_code: Optional[str] = Field(
```
Comment (amit-chandak-unskript) [inline AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py:10]: @dougsillars shouldnt service_code and quota_code be an enum?

Comment (dougsillars) [reply AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py:10]: no. There are thousands of quota codes, and new ones added on the regular.  If we force enum values, our Action will be out of date as soon as we publish it,

Comment (amit-chandak-unskript) [reply AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py:10]: How easy is it for someone to get these quota codes? Is it on webpage somewhere? We should put that link in the description then. @dougsillars

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**